app/src/main/res/layout/activity_main.xml

```diff
@@ -6,18 +6,63 @@
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         tools:context=".MainActivity">
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:orientation="vertical" >
+
+            <TextView
+                android:id="@+id/main_text"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_margin="5sp"
+                android:rotationX="0"
+                android:text="@string/Title"
+                android:textAlignment="center"
+                style="@style/Text.Title"/>
+
+            <TextView
+                android:id="@+id/departureLabel"
+                android:text="@string/departureLabel"
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                style="@style/Text.Subtitle"
+                />
+
+            <Spinner
+                android:id="@+id/departureSpinner"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"/>
+
+            <TextView
+                android:id="@+id/arrivalLabel"
+                android:text="@string/arrivalLabel"
+                android:layout_height="wrap_content"
+                android:layout_width="wrap_content"
+                style="@style/Text.Subtitle"
+                />
+
+            <Spinner
+                android:id="@+id/arrivalSpinner"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"/>
+
+            <Button
+                android:text="@string/submitButton"
+                android:textAllCaps="false"
```

Comment:
What happens if we don't have this line?

Reply:
For some reason the button text is set to "all caps" as default. Maybe not use a "<Button"?

Reply:
Yeah maybe change to use a different component